modules/xfem/src/efa/EFAElement3D.C

```diff
@@ -108,6 +139,106 @@ EFAElement3D::~EFAElement3D()
   }
 }
 
+void
+EFAElement3D::setLocalCoordinates(unsigned int num_faces)
+{
+  if (num_faces == 6)
+  {
+    /*
+    HEX27(HEX20):  7              18             6
+                   o--------------o--------------o
+                  /:             /              /|
+                 / :            /              / |
+                /  :           /              /  |
+             19/   :        25/            17/   |
+              o--------------o--------------o    |
+             /     :        /              /|    |
+            /    15o       /    23o       / |  14o
+           /       :      /              /  |   /|
+          4/        :   16/             5/   |  / |
+         o--------------o--------------o    | /  |
+         |         :    |   26         |    |/   |
+         |  24o    :    |    o         |  22o    |
+         |         :    |       10     |   /|    |
+         |        3o....|.........o....|../.|....o
+         |        .     |              | /  |   / 2
+         |       .    21|            13|/   |  /
+      12 o--------------o--------------o    | /
+         |     .        |              |    |/
+         |  11o         | 20o          |    o
+         |   .          |              |   / 9
+         |  .           |              |  /
+         | .            |              | /
+         |.             |              |/
+         o--------------o--------------o
+         0              8              1
+
+     */
+    _local_node_coor.push_back(EFAPoint(0.0, 0.0, 0.0));
```

Comment:
Also, you may as well only size these for the number of nodes you actually have (20 rather than 27 for HEX20).